traceapp/dashboard.go

```diff
@@ -136,6 +112,13 @@ func (a *App) serveDashboardData(w http.ResponseWriter, r *http.Request) error {
 		end = basis.Add(time.Duration(v) * time.Hour)
 	}
 
+	traces, err := a.Queryer.Traces(appdash.TracesOpts{
+		Timespan: appdash.Timespan{S: start, E: end},
+	})
+	if err != nil {
+		return err
+	}
```

Comment:
There is a rather fatal issue here. `InfluxDBStore.Traces` only returns e.g. 10 traces at a time due to pagination. So it right now returns "traces in the last 72 hr, maximum 10" but ideally the dashboard shows us N slowest traces over the past 72hr (all traces, not just 10).

I think this also makes the reported numbers on the dashboard less accurate, because it would be the average time of 10 traces, instead of the average time of all traces in the selected timeline.

I tried lifting the limit on `InfluxDBStore.Traces` so it returns all traces, but if there are many the query becomes extremely slow (obviously). We will need to use InfluxDB's average etc features instead.